### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by entaytion** [Zafros56/NitroGiftGeneratorAndChecker#1]
Hello. Do you plan to clean up non-working codes from the text and create a separate text document for working nitro?

**Comment by Zafros56** [comment]
Yes, I plan to make a separate file for working nitro, but now I don't have time for that.  Thanks, goodbye

**Comment by entaytion** [comment]
Thank you. Another idea might be for the proposal to add the ability to select the type of generator: classic or boost.
